.github/dependabot.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  # Docker images
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
why daily and not on based on PR merge?

Reply:
dependabot will automatically look for update and create PRs to update our dockerfile. So it's better to have it daily.  It's not tied to the contribution process.